alephium/contracts/token_bridge_scripts/complete_transfer.ral

```diff
@@ -1,7 +1,7 @@
 TxScript CompleteTransfer(
+    bridgeRewardRouter: BridgeRewardRouter,
     tokenBridgeForChain: TokenBridgeForChain,
     vaa: ByteVec
 ) {
-    let caller = callerAddress!()
-    tokenBridgeForChain.completeTransfer{caller -> ALPH: dustAmount!()}(vaa, caller)
+    bridgeRewardRouter.completeTransfer{callerAddress!() -> ALPH: dustAmount!()}(tokenBridgeForChain, vaa, callerAddress!())
```

Comment:
We should have two versions: the old `CompleteTransfer` and a new `CompleteTransferWithReward`. We will need to switch back to `CompleteTransfer` after the bootstrap phase.